src/utilities/hydration.js

```diff
@@ -2,8 +2,9 @@ export const rehydrate = function(obj) {
   try {
     return JSON.parse(new Buffer(obj, 'base64').toString());
   } catch (e) {
-    alert('Unable to parse values passed in url query parameters');
-    console.error(e);
+    if (typeof alert !== 'undefined') {
```

Comment:
The thing that prompted me to remove the console.error was that it would show up in the tests. This is also an error that can be caused by users and I would like the laboratory to recover after the user pastes corrupt data into the URL.

How should I proceed?